### ISSUE TITLE
Need advice on training Russian model

### ISSUE THREAD

**Issue by Nistrian** [TensorSpeech/TensorFlowTTS#412]
Hello! I am very impressed with your work, it provides me with many opportunities and I rely on it a lot. Some time ago I asked questions about solving some problems when training a model in Russian. I overcame all these difficulties and trained the model using mfa. However, the results were disappointing. The model produces speech that is really similar to Russian, but it cannot be called "good". Speech quality is not at all similar to that obtained with a pretrained English model. There is a big problem with emphasis. Since I have very little knowledge in this area, I would like to receive advice on what features should be taken into account when teaching the Russian-speaking model. If there are people who have been able to teach a good Russian model, I would be very grateful for the help.

**Comment by dathudeptrai** [comment]
> I have very little knowledge in this area, I would like to receive advice on what features should be taken into account when teaching the Russian-speaking model. If there are people who have been able to teach a good Russian model, I would be very grateful for the help.

hi, can you provide some information about ur dataset (hz, number of sample, multi-speaker or not, ...) and what you did.

**Comment by Nistrian** [comment]
My dataset consists of 25 hours of recording from one speaker. Sample length from 3 to 10 seconds, 22050 Hz.
I followed the scripts described in examples/mfa_extraction

Initially, I extracted the durations by specifying the configuration for one speaker (examples/fastspeech2/conf/fastspeech2.v1.yaml), as well as specifying the type of dataset ljspeech, which entailed an error with the dimensions. Later, I followed the entire extraction algorithm exactly as described in the readme(1,2,4,5 points). To do this, I changed the transcription to the form: name_file | text | name_speaker
The speaker's name was the same everywhere. All this allowed me to train a model with the characteristics described earlier. Here are the training schedules:

![image](https://user-images.githubusercontent.com/69087388/101521649-2cd60300-3997-11eb-86bd-4b517e998cb3.png)
